libraries/AP_Motors/AP_MotorsTri.cpp

```diff
@@ -64,11 +64,7 @@ void AP_MotorsTri::init(motor_frame_class frame_class, motor_frame_type frame_ty
 void AP_MotorsTri::set_frame_class_and_type(motor_frame_class frame_class, motor_frame_type frame_type)
 {
     // check for reverse tricopter
-    if (frame_type == MOTOR_FRAME_TYPE_PLUSREV) {
-        _pitch_reversed = true;
-    } else {
-        _pitch_reversed = false;
-    }
+    _pitch_reversed = frame_type == MOTOR_FRAME_TYPE_PLUSREV;
```

Comment:
This one needs prens.

Reply:
yes, I'd like us to use brackets in any case where the return has a logic operation.

So for example, there's another line that looks like this:

> return hal.gpio->read(_pin_deployed) == _pin_deployed_polarity;

but I think this should be

>> return (hal.gpio->read(_pin_deployed) == _pin_deployed_polarity);

Reply:
Sadly there is no much to say here besides that Ardupilot does not have a standard code style and also is not consistent, so I'm keen to agree with personal style changes that are no documented or enforced by the project code style, tools such as clang-format or even astyle should be a **must** in the project to avoid such things in the PRs.
At my point of view, I really don't care, but code standards does not exist to make people happy, but to avoid unnecessary discussions. 
![image](https://user-images.githubusercontent.com/1215497/107058860-e42ea080-67b3-11eb-9b79-80efa515e079.png)